.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Thank you for pull request.
+
+Below are a few things we ask you kindly to self-check before getting a review.
+
+* [ ] Used a fork of the feedstock to propose changes
+* [ ] Bump the build number (if the version is unchanged)
+* [ ] Reset the build number to `0` (if the version changed)
+* [ ] [Re-render]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
+* [ ] Updating pinnings based on the [current list]( https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L39 )
```

Comment:
@jakirkham, we need to remove this in conda-smithy

Reply:
Fixing with PR ( https://github.com/conda-forge/conda-smithy/pull/683 ).